src/Common/test-runtime/project.json

```diff
@@ -63,7 +63,7 @@
     "frameworks": {
        "dnxcore50": { 
          "dependencies": {
-           "System.Net.Http.WinHttpHandler": "4.0.0-rc2-23602",
+           "System.Net.Http.WinHttpHandler": "4.0.0-rc2-23608",
            "System.Net.Security": "4.0.0-rc2-*"
```

Comment:
Crap, I missed the System.Net.Security change last time. Can you update this please?